test/libsolidity/syntaxTests/types/weird_sized_types.sol

```diff
@@ -0,0 +1,41 @@
+contract C {
+    bytes0 b1;
+    bytes01 b2;
+    bytes000000001 b3;
+    bytes000099000 b4;
+    bytes0a b5;
+
+    int0 i1;
+    int01 i2;
+    int000000001 i3;
+    int000099000 i4;
+    int0a i5;
+
+    uint0 u1;
+    uint01 u2;
+    uint000000001 u3;
+    uint000099000 u4;
+    uint0a u5;
+
+    fixed0x0 f1;
+    fixed01x1 f2;
+    fixed1x01 f3;
+    fixed000000001x1 f4;
+    fixed1x000000001 f5;
+    fixed000099000x1 f6;
+    fixed1x000099000 f7;
+    fixed0ax1 f8;
+    fixed1x0a f9;
+
+    ufixed0x0 g1;
+    ufixed01x1 g2;
+    ufixed1x01 g3;
+    ufixed000000001x1 g4;
+    ufixed1x000000001 g5;
+    ufixed000099000x1 g6;
+    ufixed1x000099000 g7;
+    ufixed0ax1 g8;
+    ufixed1x0a g9;
```

Comment:
```suggestion	uint bytes0 = 0;
	uint bytes01 = 0;
	uint bytes000000001 = 0;
	uint bytes000099000 = 0;
	uint bytes0a = 0;
	
	uint int0 = 0;
	uint int01 = 0;
	uint int000000001 = 0;
	uint int000099000 = 0;
	uint int0a = 0;
	
	uint uint0 = 0;
	uint uint01 = 0;
	uint uint000000001 = 0;
	uint uint000099000 = 0;
	uint uint0a = 0;
	
	uint fixed0x0 = 0;
	uint fixed01x1 = 0;
	uint fixed1x01 = 0;
	uint fixed000000001x1 = 0;
	uint fixed1x000000001 = 0;
	uint fixed000099000x1 = 0;
	uint fixed1x000099000 = 0;
	uint fixed0ax1 = 0;
	uint fixed1x0a = 0;
	
	uint ufixed0x0 = 0;
	uint ufixed01x1 = 0;
	uint ufixed1x01 = 0;
	uint ufixed000000001x1 = 0;
	uint ufixed1x000000001 = 0;
	uint ufixed000099000x1 = 0;
	uint ufixed1x000099000 = 0;
	uint ufixed0ax1 = 0;
	uint ufixed1x0a = 0;
```

That'd also be a way to test this, wouldn't it :-)?

Reply:
Agree.

Reply:
Yeah I guess post-change. I created the test pre-change, because wanted to see proper existing errors. However due to `DeclarationError` being fatal it wasn't possible. Hence I mentioned it in https://github.com/ethereum/solidity/pull/10580#discussion_r541122645, but you guys just jumped the gun :)